recipes-extended/xen/xen.bb

```diff
@@ -85,6 +85,10 @@ FILES_${PN}-xen-shim = "\
     ${libdir}/xen/boot/xen-shim \
     "
 
+RDEPENDS_${PN}-scripts-block += " \
```

Comment:
I have a couple concerns with this...
First, the scripts included in xen-scripts-block don't appear to use perl. Did you mean `${PN}-scripts-common`? That one includes `locking.sh`, which uses perl once.
The second thing is, do we really want to add perl to dom0 for just one call in one script? I would be against that, unless it's absolutely irreplaceable for some reason... How about a patch to replace that one call with something else? That patch could be submitted upstream, and ultimately removed from OpenXT, without ever having to include perl in dom0.

Reply:
I was aware of minimal use of `perl` but that type of decision is out of my hands.  @eric-ch what do you think?

Reply:
I may have a replacement for Perl using a new little program. Needs some testing, but hopefully I can post it in the next day or two.

Reply:
This is what I'm thinking:  https://github.com/jandryuk/xenclient-oe/commit/b9f4294cb4b213d8391f69b738e9283bda14d825

I've only tested cmp-fd-file-inode outside of Xen at this time.

Reply:
@jandryuk I think that makes sense. The `perl` runtime dependency adds around 2.2M (per `packages-split` on `perl` and `perl-lib` its runtime dependency).

If you plan to PR your new tool, do you mind this PR being merged first then yours will remove the `perl` `RDEPENDS`?

Reply:
cmp-fd-file-inode isn't ready to go yet (looks like I should put it into xen-libxl).  So you can go ahead with this PR and mine will come along later.

Reply:
@jandryuk do you still plan on PR-ing the perl removal? Knowing how many "temporary" fixes became very much permanent in OpenXT, I'm uneasy about merging this without being sure that the proper fix is on its way! Thank you

Reply:
I have a branch.  I'll give it a quick spin against the latest version of this PR.

Reply:
Perl removal posted here: https://github.com/OpenXT/xenclient-oe/pull/1268